### PR TITLE
fix: update `bee-metrics` and log first error for process metrics

### PR DESCRIPTION
**Pull request by pvdrz**

### CODE REVIEW

File: bin/inx-chronicle/src/metrics.rs [hunk 1]
```diff
@@ -89,9 +89,13 @@ impl Actor for MetricsWorker {
 
         let process_metrics_handle = {
             tokio::spawn(async move {
+                if let Err(e) = metrics.update().await {
```
Comment (Alexandcoats) [inline bin/inx-chronicle/src/metrics.rs:92]: Shouldn't the task just exit in this case?

Comment (pvdrz) [reply bin/inx-chronicle/src/metrics.rs:92]: Not sure how likely is to have a one time error that never repeats like `proc` files being unavailable. To be honest I never expected this to fail at all.

Comment (Alexandcoats) [reply bin/inx-chronicle/src/metrics.rs:92]: I'm confused. If this fails the first time, won't it just fail over and over in the loop?

Comment (pvdrz) [reply bin/inx-chronicle/src/metrics.rs:92]: I am not sure. I'd believe that it is possible that `update` could fail the first time and not fail later but I don't have any scenario to prove it either.

**Review by Alexandcoats — DISMISSED** [review]

File: bin/inx-chronicle/src/metrics.rs [hunk 2]
```diff
@@ -89,10 +89,24 @@ impl Actor for MetricsWorker {
...
+                            retries = MAX_RETRIES;
+                        }
+                        Err(e) => {
+                            log::warn!("Cannot update process metrics: {e}");
+                            retries -= 1;
+                        }
+                    }
                 }
+
+                log::warn!("Could not update process metrics after {MAX_RETRIES} retries");
```
Comment (grtlr) [inline bin/inx-chronicle/src/metrics.rs:109]:
```suggestion
                log::warn!("Could not update process metrics after {MAX_RETRIES} retries.");
```

**Review by grtlr — DISMISSED** [review]

**Review by Alexandcoats — DISMISSED** [review]

**Review by grtlr — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**